tasks/collect-rh-sign-image-params/collect-rh-sign-image-params.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-rh-sign-image-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/tags: release
+spec:
+  params:
+    - description: An image of operator-pipeline-images for the steps to run in.
+      name: pipeline_image
+      default: "quay.io/redhat-isv/operator-pipelines-images:released"
+    - name: config_map_name
+      description: Name of a configmap with pipeline configuration
+  results:
+    - name: pyxis_url
+      description: Container API URL based for selected environment
+    - name: sig_key_id
+      description: The signing key id that index image claims are signed with
+    - name: sig_key_name
+      description: The signing key name that index image claims are signed with
+    - name: umb_url
+      description: umb host to connect to for messaging, e.g. for signing
+    - name: umb_listen_topic
+      description: umb topic which is used for listening
+    - name: umb_publish_topic
+      description: umb topic which is used for publishing
+    - name: umb_client_name
+      description: Client name to connect to umb, usually a service account name
+    - name: ssl_cert_secret_name
+      description: SSL secret name
+    - name: ssl_cert_file_name
+      description: SSL certificate file name
+    - name: ssl_key_file_name
+      description: SSL key file name
+  steps:
+    - name: collect-rh-sign-image-params
+      image: "$(params.pipeline_image)"
+      env:
+        - name: config_map_name
+          value: $(params.config_map_name)
+      script: |
+        #!/bin/bash
+        set -ex
+
+        configMapJson=$(oc get "cm/${config_map_name:?}" -ojson)
+        PYXIS_URL=$(jq -r '.data.PYXIS_URL' <<< "${configMapJson}")
```

Comment:
Should any/all of these have `-e` so the task fails if they aren't present? I see none of defaults so I am thinking probably